cpp/src/slice2js/Gen.h

```diff
@@ -31,7 +28,7 @@ namespace Slice
 
         ::IceInternal::Output& _out;
 
-        std::vector<std::pair<std::string, std::string>> _imports;
```

Comment:
Since we never pass `_imports` into the constructor, we may as well just default initialize it like this.